jquery.sumoselect.js

```diff
@@ -136,6 +137,12 @@
 
                 prepItems: function (opts, d) {
                     var lis = [], O = this;
+                    if(settings.selectedFirst)
+                        opts.sort(function(a,b){
+                            if(a.selected && !b.selected) return -1;
+                            if(b.selected && !a.selected) return 1;
+                            return 0;
+                        });

```

Comment:
This ordering only works at page load or after reloading the plugin.  
I'll merge it if you add the ordering on select/deselect too. This is too confusing for users otherwise